### DISCUSSION TITLE
Adding time metadata : conversion to UTC

### DISCUSSION THREAD

**Discussion by ymelnikova** [biigle/core#951] [Q&A] [answered]
Hi team, 
I noticed that when I add metadata ['taken_at] , the time is converted to UTC ( as it is default for Biigle) by substracting 2 hours in my case. But how the current time zone is defined to calculate this conversion? Is it defined by where the biigle server is hosted?  Thanks.

**Comment by mzur** [comment, marked as answer]
If you run your own BIIGLE instance, the timezone can be configured in the [build/.env](https://github.com/biigle/biigle/blob/master/build/.env.example#L10) file. Currently, the default is `Europe/Berlin`. Since you mention it, we should probably change the default to UTC (https://github.com/biigle/biigle/pull/31). The timezone of your instance at biigle.de is `Europe/Berlin`.

**Reply by ymelnikova** [reply]
Thank you Martin! It works now.
